web/react/utils/utils.jsx

```diff
@@ -398,10 +398,10 @@ module.exports.textToJsx = function(text, options) {
     // Function specific regexes
     var hashRegex = /^href="#[^"]+"|(#[A-Za-z]+[A-Za-z0-9_]*[A-Za-z0-9])$/g;
 
-    var implicitKeywords = {};
+    var implicitKeywords = [];
     var keywordArray = UserStore.getCurrentMentionKeys();
     for (var i = 0; i < keywordArray.length; i++) {
-        implicitKeywords[keywordArray[i]] = true;
+        implicitKeywords[i] = keywordArray[i];
```

Comment:
Doesn't this loop just make implicitKeywords the same as keywordArray now?

Should just be able to do this I think
```
var implicitKeywords = UserStore.getCurrentMentionKeys();
```